package.json

```diff
@@ -47,5 +52,10 @@
     "*.{js,jsx,ts,tsx}": "eslint --cache --fix",
     "*.{ts,tsx,js,jsx,css,md,mdx}": "prettier --write"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@inrupt/solid-client-authn-core": "^1.2.4",
```

Comment:
I don't think this would ever need to be a direct dependency for consumers?

Reply:
Excellent point, I forgot that was the case. Fixed in 71d335687c7f9606fc31c295663262ce4d72a113